lib/manager/components/version/FeedVersionSpanChart.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+
+import React, { Component } from 'react'
+import { Table } from 'react-bootstrap'
+import moment from 'moment'
+
+import type { FeedVersion } from '../../../types'
+
+type Props = {
+  activeVersion: FeedVersion,
+  comparedVersion: FeedVersion,
+  comparedVersionIndex: number
+}
+
+const dateFormat = 'MMM. DD, YYYY'
+
+/** Renders the validity span of one feed using the given props. */
+const FeedSpan = ({
```

Comment:
Can this be put after the `FeedVersionSpanChart` component so that there is no confusion about which component the `Props` belong to?

Reply:
Updated in https://github.com/ibi-group/datatools-ui/pull/528/commits/08c881ccaf552b503dddbb05ada677e37e3a9272.